### ISSUE TITLE
Network Error when downloading from browser

### ISSUE THREAD

**Issue by Mcad30** [barryvdh/laravel-dompdf#982]
**This is just a Dompdf wrapper!**
I understand that this package is just a Laravel wrapper for https://github.com/dompdf/dompdf
Any issues with PDF rendering, CSS that is not applied correctly, aligning/fonts/characters etc that are not directly related to this package, should be reported there. When having doubts, please try to reproduce the issue with just dompdf. If it's also present there, do not open an issue here please.

**Describe the bug**
First of all, thank you for making this package for Laravel.

I've successfully converted an html view file into a PDF format. However, when I try clicking my view button which redirects me to a browser-view of the html , and click download, the file explorer shows up asking me where I want to save the file. I noticed that the file type was "All Files (*)" and not pdf. I tried saving it but it gives me a 'Failed - Network Error'. I'm guessing it's because of the file type not being detected as pdf? Like I should set the content type of the file? idk.

I'm not sure if it's meant to be like that or I'm doing something wrong. 

**To Reproduce**
- Laravel and package version
Package version: v2.0.0
Laravel version: 9.5.1
- Input HTML/CSS

Here's my code:
- To return the converted HTML to PDF file:

        $html = view('print.test_result', $data)->render();
        return PDF::loadHTML($html, 'utf-8')->setPaper('Short', 'portrait')->stream();

**Expected behavior**
I was expecting to have the pdf file type in the choices.

**Screenshots**
The first image is the browser's view of the converted HTML to PDF file where I click the download button.
The second image is the file explorer where I select the path where I want to save the file.
The third image is where I clicked save and received a 'Failed - Network error'
![1](https://github.com/barryvdh/laravel-dompdf/assets/102274600/37b82924-8502-4679-aeed-1fa9031c511b)
![2](https://github.com/barryvdh/laravel-dompdf/assets/102274600/fde01323-b0b8-448e-b211-d55a495a1ed4)
![3](https://github.com/barryvdh/laravel-dompdf/assets/102274600/ea3a6391-8ca8-484b-8cdd-99d0c559a597)

**Additional context**

**Comment by parallels999** [comment]
```php
return PDF::loadView('print.test_result', $data)->setPaper('Short', 'portrait')->stream();
```

**Comment by Mcad30** [comment]
> ```
> use Barryvdh\DomPDF\Facade\Pdf;
> use Illuminate\Support\Facades\Response;
> 
> ///
> 
> $name = 'file_name.pdf'
> return Response::make(
>     Pdf::loadView('print.test_result', $data)->setPaper('Short', 'portrait')->output(),
>     200,
>     [
>         'Content-Type' => 'application/pdf',
>         'Content-Disposition' => 'inline; filename="'.$name.'"'
>     ]
> );
> ```

Thank you for taking the time to help.

I seem to be getting the same problem after I pasted your suggestion.

I'm also able to see the PDF converted but can't seem to download it still.

**Comment by Mcad30** [comment]
Oh actually I found a workaround. Not code related though. I'll just show it how in case someone might find it useful.

So instead of saving it right away, I clicked the print icon and clicked save as PDF.

I was able to save it as a pdf file without having the Network Error.

Thank you  @parallels999 for your help :)

![1](https://github.com/barryvdh/laravel-dompdf/assets/102274600/d734947f-17ce-4cee-8151-ff1d4cd17fb9)
![2](https://github.com/barryvdh/laravel-dompdf/assets/102274600/ca8537e6-12ba-4415-b5b3-0ad4a7c41c71)
